.env.example

```diff
@@ -1,6 +1,6 @@
 APP_NAME=Laravel
 APP_ENV=local
-APP_KEY=
+APP_KEY=base64:32UdkmCHGq6/Vi38PqX2C7HXIVMJG6b5ok24N1ufRQQ=
```

Comment:
.env.example shouldn't be touched unless adding new variables

Reply:
Thanks!!! A LOT!

Reply:
Watch Povilas video, in the beginning he talks about env
https://www.youtube.com/watch?v=0ieY-YmtGKA